frontend/test/__support__/e2e/commands.js

```diff
@@ -1,310 +1,18 @@
-import { USERS } from "__support__/e2e/cypress_data";
+import "./commands/ui/button";
+import "./commands/ui/icon";
 
-Cypress.Commands.add("createUser", user => {
-  cy.log(`Create ${user} user`);
-  return cy.request("POST", "/api/user", USERS[user]).then(({ body }) => {
-    // Dismiss `it's ok to play around` modal for the created user
-    cy.request("PUT", `/api/user/${body.id}/qbnewb`, {});
-  });
-});
+import "./commands/api/question";
+import "./commands/api/dashboard";
 
-Cypress.Commands.add("signIn", (user = "admin") => {
-  const { email: username, password } = USERS[user];
-  cy.log(`Logging in as ${user}`);
-  cy.request("POST", "/api/session", { username, password });
-});
+import "./commands/user/createUser";
+import "./commands/user/authentication";
 
-Cypress.Commands.add("signInAsAdmin", () => {
-  cy.signIn("admin");
-});
+import "./commands/permissions/updatePermissions";
+import "./commands/permissions/sandboxTable";
 
-Cypress.Commands.add("signInAsNormalUser", () => {
-  cy.signIn("normal");
-});
+import "./commands/database/addH2SampleDataset";
 
-Cypress.Commands.add("signInAsSandboxedUser", () => {
-  cy.signIn("sandboxed");
-});
+import "./commands/helpers/isVisibleInPopover";
+import "./commands/helpers/isInViewport";
```

Comment:
```suggestion
import "./commands/visibility/isVisibleInPopover";
import "./commands/visibility/isVisibleInWindow";
```

Here we could consider two changes for clarity, if I understood .

Assumption: both `isVisibleInPopover` and `isInViewport` are about _checking visibility_,

1.  maybe the `helpers` dir could be renamed `visibility` so we know more immediately what these files are about.

2. maybe `isInViewport` could be renamed `isVisibleInWindow` for consistency with `isVisibleInPopover`.

2.

Reply:
Helpers might (and probably will) contain commands that are not related to the visibility. Is nesting the folder structure one more level deep too bad?
```javascript
import "./commands/helpers/foo/bar";
import "./commands/helpers/visibility/isInViewport";
```

Reply:
Also, left some more questions in the original description. Please, take a look.

Reply:
> I could be wrong about the goal of this command...because a mere findBy or get should do the same, but just to double check.

The visibility for Cypress and for the user are two completely different things. Cypress "sees" anything that is in DOM and considers it visible. Wit this helper function we want to check is an element literary visible within the boundaries of the current viewport without having to scroll. It's handy for detecting when our rendering (especially of the popover) goes haywire and we lose that element.

Reply:
> Helpers might (and probably will) contain commands that are not related to the visibility. Is nesting the folder structure one more level deep too bad?

At the moment, these two commands we have are about visibility.

So having `helpers/visibility` looks like optimizing for something we don't have, which are other kinds of helpers.

We don't seem to be mapping first level directories with one kind of thing (say features, or components, as we have both `ui` and `database` as first level directories. And this seems ok. So would it seem ok to me if we have simply `./commands/visibility`.